core/src/main/java/tachyon/worker/netty/NettyDataServer.java

```diff
@@ -115,7 +114,7 @@ public boolean isClosed() {
    * Creates a default {@link io.netty.bootstrap.ServerBootstrap} where the channel and groups are
    * preset. Current channel type supported are nio and epoll.
    */
-  private ServerBootstrap setupGroups(final ServerBootstrap boot, final ChannelType type) {
+  private void setupGroups(final ServerBootstrap boot, final ChannelType type) {
```

Comment:
The previous way takes an input, makes change inside and returns this as output, which is just a bit wierd to me. How about the attemp in my new commit by creating boot inisde the method? 

if you don't like it, I will just revert it back to the previous way 
(i.e., ServerBootstrap setupGroups(final ServerBootstrap boot, final ChannelType type)).

Thanks.